cmd/admin-revoker/main.go

```diff
@@ -163,15 +164,11 @@ func main() {
 				cac, auditlogger, dbMap, _, _ := setupContext(c)
 
 				tx, err := dbMap.Begin()
-				if err != nil {
-					tx.Rollback()
-				}
+				err = probs.WithRollbackError(tx, err)
```

Comment:
I think this would be clearer as:

```
if err != nil {
  cmd.FailOnError(probs.WithRollbackError(tx, err), ...)
}
```

However, it's probably even clearer to move the `WithRollbackError` to the SA package and just call it `Rollback`, so you'd have:

```
if err != nil {
  cmd.FailOnError(sa.Rollback(tx, err), "...")
}
```